src/routes/signin.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+	import Input from '$lib/forms/Input.svelte';
+	import Button from '$lib/button/Button.svelte';
+	import { auth } from '$modules/firebase/firebase';
+	import { signInWithEmailAndPassword } from 'firebase/auth';
+
+	let email = '';
+	let password = '';
+
+	const signIn = () => {
+		signInWithEmailAndPassword(auth, email, password)
+			.then(async (userCredential) => {
+				const user = userCredential.user;
+				console.log(user);
+				// TODO: redirect to maypage
+			})
+			.catch((error) => {
+				console.log(error);
```

Comment:
めっちゃ細かいけど
```suggestion
				console.error(error);
```
とかの方が正しい気がします

Reply:
そのようにしました